build.sbt

```diff
@@ -330,7 +330,8 @@ lazy val docs = project
       (ThisBuild / baseDirectory).value / "website" / "target" / "docs",
     mdocExtraArguments := List("--no-link-hygiene"),
     mdocVariables := Map(
-      "VERSION" -> version.value.replaceFirst("\\+.*", "")
+      "VERSION" -> version.value.replaceFirst("\\+.*", ""),
+      "STABLE_VERSION" -> "0.7.29"
```

Comment:
I would've probably left `VERSION` as-is, and introduced a `NEXT_MAJOR_VERSION` or something, so that you wouldn't have to change existing uses of `VERSION`, but it's a minor thing, feel free to ignore